precompiles/assets-erc20/Roles.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity >=0.8.3;
+
+/// @title Extension of the ERC20 interface that allows users to
+/// get the account capable of fullfiling different asset roles
```

Comment:
We definitely need the `@author` tag, maybe you can adapt the others (if/where applicable). As an example:

```suggestion
/// @author The Moonbeam Team
/// @title ERC20 interface Asset Roles
/// @dev Extension of the ERC20 interface that allows users to get the account capable of fulfilling different asset roles
/// @custom:address 0x0000000000000000000000000000000000000XXX
/// @title 
```